test/tests/functional/pbs_ralter.py

```diff
@@ -1156,23 +1168,22 @@ def test_alter_standing_resv_boundary_values(self):
     def test_alter_degraded_resv_mom_down(self):
         """
         This test checks the alter of start and end times of reservations
-        when mom is down
+        when mom is down.
         """
         duration = 30
         shift = 5
-        offset = 5
+        offset = 60
```

Comment:
Why the increase in offset?

Reply:
To avoid the race condition. Since I am bringing down the Mom. In slow machines, small offset can cause issue if there are delays in Mom coming up. So increased the offset. Anyways the offset value is not going to change the execution time of the test case.